HTTPlayground/base.py

```diff
@@ -0,0 +1,46 @@
+from http.cookies import SimpleCookie
+from pathlib import Path
+from HTTPlayground.settings import BASE_DIR, TEMPLATE_DIR, STATIC_URL
+
+
+def send_headers(request, response_code: int = 200, content_type: str = 'text/plain', **kwargs):
+    request.send_response(response_code)
+    request.send_header('Content-type', content_type)
+    for k, v in kwargs.items():
+        request.send_header(k, v)
+    request.end_headers()
+
+
+class CookieHandler:
+    @staticmethod
+    def is_cookie(request, cookie_pair: dict):
```

Comment:
minor: `is_cookie` name is not meaningful. Smth like `has_cookie` would make more sense (like request has the cookie with this value).